test-files/cmd/quick/HelpCmdVerboseTest.sh.stdout

```diff
@@ -92,7 +92,7 @@ The `alpha-shape` command provides a way to generate a concave hull for a set of
 * `--buffer` - Optional buffer, in meters, to add to the alpha shape before cutting. A negative value will make the 
                shape smaller. If using the alpha shape command with the `cut` command, it is helpful to use a larger 
                negative value on complex data shapes to limit the number of holes in the output data. A negative value 
-               equal to the alpha value is a good rule of thumb.
+               equal to the alpha value is a good general rule.
```

Comment:
This is test output that gets compared (though the test is disabled).  Best not to change it.

Reply:
It would be a good idea to change it because it has misogynistic connotations.